devicon.json

```diff
@@ -1282,6 +1416,9 @@
     },
     {
         "name": "debian",
+        "altnames": [
+            "debian"
```

Comment:
`debian` is already the name, no need for an altname imo.

Reply:
3c5bf839850823623296b0f0a222ba65aff6386c